core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -194,7 +194,7 @@ public OutputStream getOutStream(long blockId, long blockSize, WorkerNetAddress
     }
     // No specified location to write to.
     if (address == null) {
-      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage());
+      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage(blockSize));
```

Comment:
The final result is "There is no worker with enough space for a new block of size 536,870,912".  Why is not 512MB?